src/content/docs/cheerp/11-guides/10-build-systems/00-make.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Make
+description: How to use Make with Cheerp
+---
+
+import LinkButton from "@/components/LinkButton.astro";
+
+To integrate Cheerp into a project that uses [Make](https://gnu.org/software/make/), use this snippet in your Makefile:
+
+```makefile
+ifeq ($(OS),Windows_NT)
```

Comment:
This is ok for  a new project, but for an existing makefile probably it makes more sense to expect to have to override CC and CXX